packit_service/worker/helpers/testing_farm.py

```diff
@@ -99,7 +99,13 @@ def skip_build(self) -> bool:
     def fmf_url(self) -> str:
         return (
             self.job_config.metadata.fmf_url
-            or self.project.get_pr(self.metadata.pr_id).source_project.get_web_url()
+            or (
+                self.metadata.pr_id
+                and self.project.get_pr(
+                    self.metadata.pr_id
+                ).source_project.get_web_url()
```

Comment:
I'd have no idea what this returns :D

Reply:
Goes from top:
1. has `fmf_url`? throws it out
2. has PR ID? throws out URL of the project from which it's being merged
3. fallbacks to current project's URL